src/main/java/uk/gov/digital/ho/hocs/info/api/dto/SchemaDto.java

```diff
@@ -4,13 +4,15 @@
 import lombok.AccessLevel;
 import lombok.AllArgsConstructor;
 import lombok.Getter;
+import lombok.NoArgsConstructor;
 import uk.gov.digital.ho.hocs.info.domain.model.Schema;
 
 import java.util.List;
 import java.util.UUID;
 import java.util.stream.Collectors;
 
-@AllArgsConstructor(access = AccessLevel.PRIVATE)
+@NoArgsConstructor
```

Comment:
Did you need to change these? It's always better to lock these down tighter hence unless you really need it, I think it's better not to change it.

Reply:
No. I've removed the change.